rhc.spec.in

```diff
@@ -33,6 +33,11 @@ BuildRequires: systemd
 rhc is a client that registers a system with RHSM and activates the Red Hat yggd
 MQTT client.
 
+%package compat
```

Comment:
This package does not require rhc nor yggdrasil to be installed. Thus, it can create symbolic link on non existing target. I think that rhc could be only recommended (https://docs.fedoraproject.org/en-US/packaging-guidelines/WeakDependencies/), but yggdrasil package should be required.

Reply:
Ah good point. I'll add those.